lib/dfe/reference_data/degrees/grades.rb

```diff
@@ -102,6 +102,14 @@ module Degrees
             suggestion_synonyms: [],
             match_synonyms: [],
             group: :main_postgrad }
+        },
+        {
+          id: :string,
+          name: :string,
+          hesa_code: :string,
+          suggestion_synonyms: { kind: :array, element_schema: :string },
+          match_synonyms: { kind: :array, element_schema: :string },
+          group: :symbol
```

Comment:
Feels unexpected to me that the schema is just tacked on at the end, as a second parameter. This might make adding new items to the list more error-prone, as they have to appear before the schema definition.

Could the schema be assigned to a constant at the top of the file maybe instead, and then referenced?

Reply:
That's a good idea, I'll do it now!